rust/index/Cargo.toml

```diff
@@ -28,13 +28,18 @@ chroma-cache = { workspace = true }
 chroma-config = { workspace = true }
 chroma-storage = { workspace = true }
 chroma-distance = { workspace = true }
+async-channel = "2.3.1"
+itertools.workspace = true
```

Comment:
can we use the same format for this as above...

Reply:
whoops yes